File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        
        i = 0
        while i < len(nums):
            j = 0
            while j < len(nums):
                if nums[i] + nums[j] == target and i!=j:
                    result = []
                    result.append(i)
                    result.append(j)
                    return result
                j+=1
            i+=1

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int len = nums.length;

        int[] result = new int[2];

        for (int i = 0; i < len; i++) {
            for (int j = 0; j < len; j++) {
                if (((nums[i] + nums[j]) == target) && (i != j)) {
                    result[0] = i;
                    result[1] = j;

                }
            }
        }
        return result;
    }
}